sample-notificator-app/src/main/resources/config.properties.example

```diff
@@ -2,4 +2,9 @@ mysql.host = ...
 mysql.pass = ...
 mysql.user = ...
 mysql.db = ...
-mysql.port = ...
\ No newline at end of file
+mysql.port = ...
+services.subscriptions.password = ...
+services.subscriptions.tokengeneration.endpoint = ...
```

Comment:
```suggestion
services.tokengeneration.endpoint = ...
```

Reply:
Needs to be the same as in dependency manager

Reply:
good catch. I adjusted it in the DependencyManager